### PR TITLE
migrate kubelet's metrics/probes & metrics endpoint to metrics stability framework

### PR DESCRIPTION
**Pull request by logicalhan**
**What type of PR is this?**

/kind feature

**What this PR does / why we need it**:

This PR migrates kubelet's metrics & metrics/probes endpoints to the [metrics stability framework](https://github.com/kubernetes/enhancements/blob/master/keps/sig-instrumentation/20190605-metrics-stability-migration.md)

**Does this PR introduce a user-facing change?**:
```release-note
Kubelet metrics for /metrics and /metrics/probes are now marked as with the ALPHA stability level.
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
[KEP - Metrics Stability Migration]: https://github.com/kubernetes/enhancements/blob/master/keps/sig-instrumentation/20190605-metrics-stability-migration.md
```

### CODE REVIEW

**Comment by BenTheElder** [comment]
/uncc @kubernetes/dep-approvers 
/uncc
please re-cc when this builds 😅

File: pkg/kubelet/dockershim/metrics/metrics.go [hunk 1]
```diff
@@ -132,14 +143,13 @@ var registerMetrics sync.Once
...
 func Register() {
 	registerMetrics.Do(func() {
-		prometheus.MustRegister(DockerOperationsLatency)
-		prometheus.MustRegister(DockerOperations)
-		prometheus.MustRegister(DockerOperationsErrors)
-		prometheus.MustRegister(DockerOperationsTimeout)
-		prometheus.MustRegister(DeprecatedDockerOperationsLatency)
-		prometheus.MustRegister(DeprecatedDockerOperations)
-		prometheus.MustRegister(DeprecatedDockerOperationsErrors)
-		prometheus.MustRegister(DeprecatedDockerOperationsTimeout)
```
Comment (dashpole) [inline pkg/kubelet/dockershim/metrics/metrics.go]: Is there a reason we are not registering this anymore?

**Review by dashpole — COMMENTED** [review]
Looks mostly good.

File: pkg/kubelet/dockershim/metrics/metrics.go (hunk 1 above)
Comment (logicalhan) [reply pkg/kubelet/dockershim/metrics/metrics.go]: It’s registered in the lines below.

Comment (dashpole) [reply pkg/kubelet/dockershim/metrics/metrics.go]: I expanded the rest of the file, and still don't see `DeprecatedDockerOperationsTimeout` registered anywhere...  can you link to it?

Comment (logicalhan) [reply pkg/kubelet/dockershim/metrics/metrics.go]: ah, good catch. super odd it's not there, as i did a find and replace.

**Pull request merged**